kolibri/utils/options.py

```diff
@@ -304,135 +257,207 @@ def origin_or_port(value):
             "type": "string",
             "default": "https://kolibridataportal.learningequality.org",
-            "envvars": ("KOLIBRI_DATA_PORTAL_SYNCING_BASE_URL",),
+            "description": "URL to use as the target for data portal syncing.",
         },
     },
     "Deployment": {
         "HTTP_PORT": {
             "type": "port",
             "default": 8080,
-            "envvars": ("KOLIBRI_HTTP_PORT", "KOLIBRI_LISTEN_PORT"),
+            "envvars": (
+                "KOLIBRI_HTTP_PORT",
+                "KOLIBRI_LISTEN_PORT",
+            ),
+            "description": "Sets the port that Kolibri will serve on. This can be further overridden by command line arguments.",
+        },
+        "RUN_MODE": {
+            "type": "string",
+            "description": "Used to flag non-user Kolibri instances",
+            "skip_blank": True,
         },
-        "RUN_MODE": {"type": "string", "envvars": ("KOLIBRI_RUN_MODE",)},
         "DISABLE_PING": {
             "type": "boolean",
             "default": False,
-            "envvars": ("KOLIBRI_DISABLE_PING",),
+            "description": "Turn off the statistics pingback. This will also disable update notifications",
         },
         "URL_PATH_PREFIX": {
-            "type": "string",
+            "type": "url_prefix",
             "default": "/",
-            "envvars": ("KOLIBRI_URL_PATH_PREFIX",),
-            "clean": lambda x: x.lstrip("/").rstrip("/") + "/",
+            "description": """
+                Serve Kolibri from a subpath under the main domain. Used when serving multiple applications from
+                the same origin. This option is not heavily tested, but is provided for user convenience.
+            """,
         },
         "LANGUAGES": {
             "type": "language_list",
             "default": SUPPORTED_LANGUAGES,
-            "envvars": ("KOLIBRI_LANGUAGES",),
+            "description": """
+                The languages that this Kolibri should enable. The default is all the languages that Kolibri supports.
+                Can either be a single language code, or a comma separated list of language codes.
+            """,
         },
         "ZIP_CONTENT_ORIGIN": {
             "type": "origin_or_port",
             "default": "",
-            "envvars": ("KOLIBRI_ZIP_CONTENT_ORIGIN",),
+            "description": """
+                When running in default operation, this will default to blank, and the Kolibri
+                frontend will look for the zipcontent endpoints on the same domain as Kolibri proper
+                but using ZIP_CONTENT_PORT instead of HTTP_PORT.
+                When running behind a proxy, this value should be set to the port that the zipcontent
+                endpoint is being served on, this will be substituted for the port that Kolibri proper
+                is being served on.
+                If zipcontent is being served from a completely separate domain this can be the absolute
```

Comment:
`When zipcontent is being served from a completely separate domain, you can set an absolute origin (full protocol plus domain, e.g. 'https://myzipcontent.com/') to be used for all zipcontent origin requests.`